src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java

```diff
@@ -104,4 +104,26 @@ public void ChatTestsStaff_004_basicUserChatFails() {
     SpecialPageObject special = new SpecialPageObject();
     special.verifyPageHeader("Permissions error");
   }
+
+  @DontRun(env = {"sandbox"})
+  @Test(groups = {"ChatTestsStaff_005", "ChatStaff", "ChatTests"})
+  public void ChatTestsStaff_005_staffOptionsNotDisplayed() {
+    openChatForUser(credentials.userNameStaff, credentials.passwordStaff);
+
+    switchToWindow(1);
+    new SpecialVersionPage().open();
+    ChatPageObject chatUserStaff2 = openChatForUser(credentials.userNameStaff2, credentials.passwordStaff2);
+    chatUserStaff2.verifyStaffOptionsNotDisplayed(credentials.userNameStaff);
+  }
+
+  @DontRun(env = {"sandbox"})
+  @Test(groups = {"ChatTestsStaff_006", "ChatStaff", "ChatTests"})
+  public void ChatTestsStaff_006_staffCanNotBlockPrivateMessages() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)